src/Explorer/Menus/CommandBar/CommandBarComponentButtonFactory.tsx

```diff
@@ -56,6 +58,9 @@ export function createStaticCommandBarButtons(
     }
   };
 
+  const homeBtn = createHomeButton();
```

Comment:
This has been added for Fabric which is problematic. The Home tab should not be available there. Can we please move it into the ` if (configContext.platform !== Platform.Fabric)`block below?